src/hooks/useSwapEstimator.js

```diff
@@ -495,33 +473,63 @@ const useSwapEstimator = ({
 
   // Fetches current gas price
   const fetchGasPrice = async () => {
+    // https://docs.avax.network/learn/platform-overview/transaction-fees/#dynamic-fee-transactions
+    let jsonCallId = 1
+    let gasPriorityFee = BigNumber.from(0)
+    const nAVAX = '000000000'
+    let gasPrice = BigNumber.from('25' + nAVAX)
+
+    const fetchMethodResult = async (method) => {
```

Comment:
Structurally, similar method is also used in accountlistener.js to fetch balances. I am not sure if this should be moved to util. Thoughts?

Reply:
Moving makes sense as this is standalone - up to you